.github/workflows/node.yml

```diff
@@ -39,14 +42,8 @@ jobs:
 
       - name: Install dependencies & build
         run: |
-          make clean
           npm ci
           npm run build --if-present
-          npm run build:package
```

Comment:
I'd like to ensure the package build does not break.
We could use a different workflow for that though.